esp32/ugfx_widgets.c

```diff
@@ -635,7 +635,13 @@ void BWgwinListDefaultDraw(GWidgetObject* gw, void* param) {
 				}
 			}
 		#endif
-		gdispGFillStringBox(gw->g.display, gw->g.x+x+LST_HORIZ_PAD, gw->g.y+y, iwidth-LST_HORIZ_PAD, iheight, qi2li->text, gw->g.font, text, fill, justifyLeft);
```

Comment:
justifyTop is [not mentioned either](http://api.ugfx.io/group___g_d_i_s_p.html#gaa56c2028ba921bcacf8bc4b843d83c85).